addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -569,13 +569,15 @@ Object.assign(MessageList, {
         hasSquashCloseMessages: false,
         haveMessagesMarkAsReadIcon: false,
         haveMessagesReplyIcon: false,
+        haveMessagesUnfollowIcon: false,
```

Comment:
This is just a forward to message, there should be no default value here if message already has one.